server/src/routes/opportunities/index.ts

```diff
@@ -250,8 +309,25 @@ export const getCVData = (_: ILogger) => async (ctx: Router.RouterContext) => {
   setResponse(ctx, OK, res);
 };
 
+export const getEditCVData = (_: ILogger) => async (ctx: Router.RouterContext) => {
+  const { githubId } = ctx.query;
+
+  const cvRepository = getRepository(CV);
+
+  const cv = await cvRepository.findOne({ where: { githubId } });
+
+  if (cv === undefined) {
+    setResponse(ctx, NOT_FOUND);
+    return;
+  }
+
+  const { id, githubId: omittedGithubId, isHidden, ...cvData } = cv;
```

Comment:
I prefer not to omit some data for such cases, but return exactly what you need. Because you can forget to add something to this omit construction in the future, and create one more security breach.

Reply:
done